txscript/scriptbuilder_test.go

```diff
@@ -7,8 +7,38 @@ package txscript
 import (
 	"bytes"
 	"testing"
+
+	"github.com/stretchr/testify/require"
 )
 
+// TestScriptBuilderAlloc tests that the pre-allocation for a script via the
+// NewScriptBuilder function works as expected.
+func TestScriptBuilderAlloc(t *testing.T) {
+	// Using the default value, we should get a script with a capacity of
```

Comment:
Might be my monitor, but looks to slightly run over line length.

Reply:
No, it's actually 3 characters short of the full 80 :sweat_smile: